src/init/update-artifacts/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.6'
 services:
 
   aeproject_node:
-    image: aeternity/aeternity:${NODE_TAG:-latest}-bundle
+    image: aeternity/aeternity:${NODE_TAG:-v6.6.0}-bundle
```

Comment:
I'd rather use v6.5.2, v6.6.0 has some memory leak as far as I know, can you make a separate PR for this?